File: programming/lab2/src/main/java/pokemons/Granbull.java
package pokemons;

import moves.physical.RockSlide;

public class Granbull extends Snubbull {
    public Granbull(String name, int level) {
        super(name, level);
        this.setStats(90, 120, 75, 60, 60, 45);
        this.addMove(new RockSlide());
    }

    public Granbull() {
        this("Granbull", 1);
    }
}

File: programming/lab2/app/src/main/kotlin/org/itmo/pokemons/Granbull.kt
package org.itmo.pokemons

import org.itmo.moves.physical.RockSlide

class Granbull(name: String, level: Int): Snubbull(name, level) {
    init {
        addMove(RockSlide())
        setStats(90.0, 120.0, 75.0, 60.0, 60.0, 45.0)
    }
}